data/transaction.go

```diff
@@ -114,4 +114,17 @@ type PreparedResults struct {
 	AlteredAccts AlteredAccountsHandler
 	Tokens       []*TokenInfo
 	TxHashStatus map[string]string
+	TxHashRefund map[string]string
+}
+
+// ResponseTransactions is the structure for the transactions response
+type ResponseTransactions struct {
+	Docs []ResponseTransactionDB `json:"docs"`
```

Comment:
use a pointer here? Like `Docs []*ResponseTransactionDB` (json unmarshaling should work)

Reply:
done.